mr/src/main/java/org/apache/iceberg/mr/mapred/IcebergWritable.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.mr.mapred;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import org.apache.hadoop.io.Writable;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.data.Record;
+
+public class IcebergWritable implements Writable {
```

Comment:
Does it make sense to move his to `iceberg-hive` module` and it can be part of the `HiveInputFormat` as described above

Reply:
The reason we didn't is because it doesn't have any Hive-specific features or imports in it. We only moved things over to `iceberg-hive` if they had Hive imports (until we ended up needing `CombineHiveInputFormat` above which broke the rule :( like you spotted above).

Reply:
Do we need an Iceberg writable? Shouldn't we use existing writables that Hive expects to receive?